python/ml4ir/applications/ranking/features/feature_fns/string.py

```diff
@@ -0,0 +1,65 @@
+import tensorflow as tf
+
+from ml4ir.base.features.feature_fns.base import BaseFeatureLayerOp
+from ml4ir.base.io.file_io import FileIO
+
+
+class QueryLength(BaseFeatureLayerOp):
+    """
+    Compute the length of the query string context feature
+    """
+    LAYER_NAME = "query_len"
```

Comment:
make it more readable
```suggestion
    LAYER_NAME = "query_length"
```